Implementation/Composing/index.md

```diff
@@ -157,7 +157,7 @@ Is an implementation of IUserComposer, that provides a quicker way to add a cust
 ```csharp
         [RuntimeLevel(MinLevel = RuntimeLevel.Run)]
```

Comment:
rather that use a `RuntimeLevel` attribute, just implement `IUserComposer` that's cleaner